net/libndp/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libndp
+PKG_VERSION:=1.6
+PKG_RELEASE:=1
+PKG_LICENSE:=LGPL-2.1
+PKG_MAINTAINER:=Thomas Guyot-Sionnest <tguyot@gmail.com>
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
```

Comment:
That's not going to fly,
Use http://libndp.org/files/libndp-1.6.tar.gz instead

Reply:
You mean fix PKG_SOURCE_URL?  I accidentally noticed I left the source alongside when committing but since I used a working template I assume it was working ok...

I didn't realize there was another place for binaries, I assume it won't need autogen neither but I will have to retest.

Reply:
We can't have unidentifiable tarballs and certainly not ones that will clash.

Reply:
The one from github is based on the tags of the official repo, so it's not "unindentifiable" and sha256 ensures it's legit... It was working, but using the prepared one (where we don't have to run autogen) is probably better... I just didn't know about it; maybe it's new as I started preparing this package about 6 month back.

Regards